components/tidb_query/src/rpn_expr/impl_math.rs

```diff
@@ -343,6 +361,95 @@ pub fn atan_2_args(arg0: &Option<Real>, arg1: &Option<Real>) -> Result<Option<Re
     })
 }
 
+#[inline]
+#[rpn_fn]
+pub fn conv(
+    n: &Option<Bytes>,
+    from_base: &Option<Int>,
+    to_base: &Option<Int>,
+) -> Result<Option<Bytes>> {
+    #[derive(Copy, Clone)]
```

Comment:
Maybe you can extract these implementations to a shared file, like `expr_util/conv.rs`? So that the vectorized and non-vectorized implementation can share same code base.

Reply:
Done.